opm/grid/cpgrid/PartitionTypeIndicator.cpp

```diff
@@ -9,18 +9,18 @@ namespace Dune
 {
 namespace cpgrid
 {
-PartitionType PartitionTypeIndicator::getPartitionType(const EntityRep<0>& cell_entity) const
+PartitionType PartitionTypeIndicator::getPartitionType(const Entity<0>& cell_entity) const
 {
     if(cell_indicator_.size())
         return PartitionType(cell_indicator_[cell_entity.index()]);
     return InteriorEntity;
 }
 
-PartitionType PartitionTypeIndicator::getPartitionType(const EntityRep<1>& face_entity) const
+PartitionType PartitionTypeIndicator::getPartitionType(const Entity<1>& face_entity) const
```

Comment:
Do we support Entity<1> for CpGrid? Don't we always use EntityRep<1> instead?

Reply:
I think we need some further refactoring to really drop that. When distributing the grid we ask for global ids on faces.

At least this change seems to work/compile.

Reply:
I see, I'm not familiar yet with global ids for faces (I did work on global ids for cells and points, and ignore faces so far). Step by step then!